src/asyncresults.jl

```diff
@@ -232,7 +232,7 @@ function async_execute(
 end
 
 function _async_execute(
-    submission_fn::Function, jl_conn::Connection; throw_error::Bool=true, kwargs...
+    submission_fn::Function, jl_conn::Connection; throw_error::Bool=true, kwargs...,
```

Comment:
Again, the trailing comma? We leave it off in an example under https://github.com/invenia/BlueStyle/#method-definitions

It doesn't really serve a purpose here and it would be a shame to hit the line limit with a useless comma.

Reply:
you are right https://github.com/invenia/LibPQ.jl/pull/197#discussion_r492908707